fuzzing/tools/launcher.py

```diff
@@ -0,0 +1,36 @@
+# Lint as: python3
+"""
+This is the launcher script to provide a uniform command line interface behind a number of arbitrary fuzzing engines.
+"""
+
+from absl import app
+from absl import flags
+from subprocess import call, TimeoutExpired
+import os
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('timeout', 20, 'test timeout.', lower_bound=0)
+flags.DEFINE_enum('engine', 'libfuzzer', ['libfuzzer'],
+                  'the fuzzing engine used to do fuzzing test.')
+
+
+def main(argv):
+    if len(argv) < 2:
+        raise app.UsageError("Too few arguments")
+
+    ret_code = 0
+    cwd = os.getcwd()
+    try:
+        # Is this the right way to execute the runnable?
+        ret_code = call(cwd + "/../../../" + argv[1], timeout=FLAGS.timeout)
+    except TimeoutExpired as e:
+        print("Error: Timeout")
+    except Exception as e:
+        print("Error: " + str(e))
+    if ret_code:
```

Comment:
Will this end up swallowing the crash and stack trace from the fuzzing call?

Reply:
Yes, it will. One way to enable the output is passing an stream to stderr and stdout in call(). I think we can add another argument for launcher.py to be the log file name?

Reply:
I double checked, if the executable is not terminated by the script, the message should be okay to be printed out to STDOUT.

Reply:
Can I mark this conversation as resolved? Or should I add a stream argument into the launcher now?

Reply:
What did you mean by "I double checked, if the executable is not terminated by the script, the message should be okay to be printed out to STDOUT."?

I think the launcher should definitely print out the error and stack trace, but also given that this is a slightly different setup that the end-goal, it's possible that it will change to do so when the script isn't run with CLI --run_under

Reply:
I mean, when the timeout argument is big enough to let the fuzzing binary stop by itself, the output will contains error and stack trace, otherwise the output only contains "Error: Timeout".

Reply:
Please see my other comment for a suggestion of a simpler alternative - using `os.exec` here.